src/components/PaymentCycleSearcher.js

```diff
@@ -0,0 +1,106 @@
+import React from 'react';
+import {
+  Searcher, useHistory, useModulesManager, useTranslations,
+} from '@openimis/fe-core';
+import { bindActionCreators } from 'redux';
+import { connect } from 'react-redux';
+import { IconButton, Tooltip } from '@material-ui/core';
+import VisibilityIcon from '@material-ui/icons/Visibility';
+import PaymentCycleFilter from './PaymentCycleFilter';
+import {
+  DEFAULT_PAGE_SIZE,
+  PAYMENT_CYCLE_ROUTE_PAYMENT_CYCLES_PAYMENT_CYCLE,
+  RIGHT_PAYMENT_CYCLE_SEARCH,
+  ROWS_PER_PAGE_OPTIONS,
+} from '../constants';
+import { fetchPaymentCycles } from '../actions';
+
+function PaymentCycleSearcher({
+  rights,
+  fetchingPaymentCycles,
+  fetchedPaymentCycles,
+  fetchPaymentCycles,
+  errorPaymentCycles,
+  paymentCycles,
+  paymentCyclesPageInfo,
+  paymentCyclesTotalCount,
+}) {
+  const history = useHistory();
+  const modulesManager = useModulesManager();
+  const { formatMessage, formatMessageWithValues } = useTranslations('paymentCycle', modulesManager);
+
+  const headers = () => [
+    'paymentCycle.year',
+    'paymentCycle.month',
+    'emptyLabel',
+  ];
+  const sorts = () => [
+    ['year', true],
+    ['month', true],
+  ];
+
+  const fetch = (params) => fetchPaymentCycles(modulesManager, params);
+
+  const rowIdentifier = (paymentCycle) => paymentCycle.id;
+  const openPaymentCycle = (paymentCycle) => history.push(
+    `/${modulesManager.getRef(PAYMENT_CYCLE_ROUTE_PAYMENT_CYCLES_PAYMENT_CYCLE)}/${paymentCycle?.uuid}`,
+  );
+
+  const itemFormatters = () => [
+    (paymentCycle) => paymentCycle.runYear,
+    (paymentCycle) => paymentCycle.runMonth,
+    (paymentCycle) => (
+      <Tooltip title={formatMessage('viewDetailsButton.tooltip')}>
```

Comment:
Seems that this translation is missing in en.json

Reply:
done